pkg/manager/member/tikv_member_manager.go

```diff
@@ -372,6 +374,29 @@ func getNewTiKVSetForTidbCluster(tc *v1alpha1.TidbCluster, cm *corev1.ConfigMap)
 		}
 	}
 
+	var additionalVolumeClaims []corev1.PersistentVolumeClaim
+	if len(tc.Spec.TiKV.StorageVolumes) > 0 {
+		for _, storageVolume := range tc.Spec.TiKV.StorageVolumes {
+			storageRequest, err := controller.ParseStorageRequest(corev1.ResourceList{
```

Comment:
No need to call `ParseStorageRequest` here, just construct the ResourceRequirements directly.